rev_news/drafts/edition-20.md

```diff
@@ -24,6 +24,46 @@ This edition covers what happened during the month of September 2016.
 
 ### Reviews
 
+* [Changing the default for "core.abbrev"?](http://public-inbox.org/git/CA+55aFy0_pwtFOYS1Tmnxipw9ZkRNCQHmoYyegO00pjMiZQfbg@mail.gmail.com/)
+
+Linus Torvalds asked for increasing the default value for number of
+characters in SHA-1 abbreviations.  The `default_abbrev = 7` was
+reasonable at early days of Git, but the project of size of the
+Linux kernel needs `git config --global core.abbrev 12`; while Git
```

Comment:
Yes using "a" here is correct (and probably `s/at early days/in the early days/`).

I also agree that a period is preferable to a semi-colon here.